doc/source/building/build_live_iso.rst

```diff
@@ -1,44 +1,45 @@
 Build an ISO Hybrid Live Image
 ==============================
 
-.. hint::
+.. sidebar:: Abstract
 
-   Make sure you have checked out the example image descriptions
-   For details see :ref:`example-descriptions`
+   This page explains how to build a live image. It contains:
 
-The following example shows how to build a live system image
-based on SUSE Leap
+   * how to build an ISO image
+   * how to start it with QEMU
```

Comment:
I found this misleading, start with qemu sounds like we give an introduction to qemu, which we don't , we just provide the command line how to start the image using qemu